README.md

```diff
@@ -1,2 +1,52 @@
-# wallet-connect-sdk
-A Wallet Connect SDK for ecosystem tools
+# Wallet Connect SDK
+A Coz's [Wallet Connect](https://walletconnect.org/) 2.0 SDK for ecosystem tools.
+
+**Wallet Connect integration is a secure solution when creating a Decentralized Application that needs user's Account
+authentication. This SDK comes in handy to help on such integration.** 
```

Comment:
**Wallet Connect integration is a secure solution for signing transactions sent to a blockchain, such as are needed when creating Decentralized Applications that require the user's account authentication. This SDK comes in handy to help on such integration.**

Reply:
Fixed